admin/class-admin.php

```diff
@@ -767,47 +767,53 @@ protected function initialize_cornerstone_content() {
 	 * @returns WPSEO_WordPress_Integration[]
 	 */
 	protected function initialize_seo_links() {
+		$integrations = array();
+
 		$link_table_compatibility_notifier = new WPSEO_Link_Compatibility_Notifier();
+		$link_table_accessible_notifier    = new WPSEO_Link_Table_Accessible_Notifier();
+
+		if ( $this->options['enable_text_link_counter'] ) {
+			$integrations[] = new WPSEO_Link_Cleanup_Transient();
+		}
+
+		if ( ! $this->options['enable_text_link_counter'] ) {
```

Comment:
Moving this conditional above the previous one, the previous one can be removed.
There is a return statement which renders the functionality executed above irrelevant.